lib/medusa/broker.ex

```diff
@@ -86,4 +85,17 @@ defmodule Medusa.Broker do
   defp route_match?(["*"|t1], [_|t2]), do: route_match?(t1, t2)
   defp route_match?([h|t1], [h|t2]), do: route_match?(t1, t2)
   defp route_match?(_, _), do: false
+
+  defp pg2_namespace, do: {:medusa, __MODULE__}
+
+  defp get_members, do: :pg2.get_members pg2_namespace
+
+  defp broadcast(pids, msg) when is_list(pids) do
```

Comment:
it would be great to have some documentation about this function and few others above as it is not totally clear what do they do.